tests/test_contentpacks.py

```diff
@@ -19,6 +19,46 @@
 logging.basicConfig()
 logging.getLogger("vcr").setLevel(logging.DEBUG)
 
+class Test_retrieve_subtitles:
+    def test_incorrect_youtube_id(self):
+        incorrect_list = ["aaa"]
+        empty_list = retrieve_subtitles(incorrect_list)
+        test_list = []
+        assert type(empty_list) == list
+        assert len(empty_list) == 0
+        assert empty_list == test_list
+
+    def test_correct_youtube_id(self):
+        correct_list = ["y2-uaPiyoxc"]
+        filled_list = retrieve_subtitles(correct_list)
+        test_list = ["y2-uaPiyoxc"]
+        assert type(filled_list) == list
+        assert len(filled_list) == 1
+        assert filled_list == test_list
+
+    def test_correct_and_incorrect_youtube_id(self):
+        mixed_list =  ["y2-uaPiyoxc", "asdadsafa"]
+        filled_list = retrieve_subtitles(mixed_list)
+        test_list = ["y2-uaPiyoxc"]
+        assert type(filled_list) == list
+        assert len(filled_list) == 1
```

Comment:
Read up on Python's truthiness idioms and then try to remove the redundancies in your assert statements.